.eslintrc.cjs

```diff
@@ -32,7 +32,7 @@ module.exports = {
         'import/no-unresolved': [
             2,
             {
-                ignore: ['@concordium/rust-bindings', 'grpc-api'],
+                ignore: ['@concordium/rust-bindings', 'grpc-api', '^#.+$'],
```

Comment:
It is not obvious why this is ignored, could you add a comment to why this was added?